testing/test_aux.out

```diff
@@ -21,20 +21,6 @@ pri.flows
 pri.ofctl
 sec.flows
 sec.ofctl
-{
```

Comment:
Why is this going away?

Reply:
That was a mistake, the new commit should have fixed it.